packages/rest/test/acceptance/file-upload/file-upload.acceptance.ts

```diff
@@ -0,0 +1,159 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/rest
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  RestBindings,
+  Request,
+  Response,
+  ControllerSpec,
+  HttpHandler,
+  SequenceActions,
+  FindRouteProvider,
+  ParseParamsProvider,
+  InvokeMethodProvider,
+  writeResultToResponse,
+  RejectProvider,
+  DefaultSequence,
+  RequestBodyParser,
+} from '../../..';
+import {inject, Context} from '@loopback/context';
+import {expect, Client, createClientForHandler} from '@loopback/testlab';
+import {RequestBodyObject} from '@loopback/openapi-v3-types';
+import {anOpenApiSpec} from '@loopback/openapi-spec-builder';
+import * as path from 'path';
+import * as multer from 'multer';
+import {createUnexpectedHttpErrorLogger} from '../../helpers';
+
+import * as express from 'express';
+
+describe('multipart/form-data', () => {
+  let client: Client;
+  beforeEach(givenHandler);
+  beforeEach(givenClient);
+  beforeEach(givenBodyParamController);
+
+  it('supports file uploads', async () => {
+    const FIXTURES = path.resolve(__dirname, '../../../../fixtures');
+    const res = await client
+      .post('/show-body')
+      .field('user', 'john')
+      .field('email', 'john@example.com')
+      .attach('certFile', path.resolve(FIXTURES, 'cert.pem'), {
+        filename: 'cert.pem',
+        contentType: 'multipart/form-data',
```

Comment:
Interesting, is the `pem` format a `multipart/form-data` too? Shouldn't we specify `application/octet-stream` instead?